articles/130_ros_time.md

```diff
@@ -39,85 +39,89 @@ Slower than real time simulation is necessary for complicated systems where accu
+
 ### Challenges in using abstracted time
 
 There are many algorithms for synchronization and they can typically achieve accuracies which are better than the latency of the network communications between devices on the network.
 However, these algorithms take advantage of assumptions about the constant and continuous nature of time.
 
-An important aspect of using an abstracted time is to be able to manipulate time.
+Again, this is not the primary intent of synchronizing time between ROS nodes. We need to ensure the ability to manipulate time.
 In some cases, speeding up, slowing down, or pausing time entirely is important for debugging.
-
-The ability to support pausing time requires that we not assume that the time values are always increasing.
+And the ability to support pausing time requires that we not assume that the time values are always increasing.
 
 When communicating the changes in time propagation, the latencies in the communication network becomes a challenge.
 Any change in the time abstraction must be communicated to the other nodes in the graph, but will be subject to normal network communication latency.
 This inaccuracy is proportional to the latency of communications and also proportional to the increase in the real time factor.
 If very accurate timestamping is required when using the time abstraction, it can be achieved by slowing down the real time factor such that the communication latency is comparatively small.
+Of course we recommend using synchronized system clock sources for live work on nodes spread across multiple devices.
 
 The final challenge is that the time abstraction must be able to jump backwards in time, a feature that is useful for log file playback.
 This behavior is similar to a system clock after a negative date change, and requires developers using the time abstraction to make sure their algorithm can deal with the discontinuity.
 Appropriate APIs must be provided for to the developer API to enable notifications of jumps in time, both forward and backwards.
 
 ### Time Abstractions
 
-There will be at least three versions of these abstractions with the following types, `SystemTime`, `SteadyTime` and `ROSTime`.
-These choices are designed to parallel the [`std::chrono`][] [`system_clock`][] and [`steady_clock`][].
+There will be at least three versions of these abstractions with the following types, `ROSTime`, `SystemTime`, and `SteadyTime`.
+The latter two choices are designed to parallel the [`std::chrono`][] [`system_clock`][] and [`steady_clock`][].
 It is expected that the default choice of time will be to use the `ROSTime` source, however the parallel implementations supporting `steady_clock` and `system_clock` will be maintained for use cases where the alternate time source is required.
 
-#### System Time
+#### Steady Time
```

Comment:
Fixed to original order. I shouldn't have reordered them. I was thinking of ordering in default -> fallback 1 -> fallback 2, but things didn't come out that way regardless.